server/chat-plugins/scavenger-games.ts

```diff
@@ -451,6 +451,53 @@ const TWISTS: { [k: string]: Twist } = {
 		},
 	},
 
+	pointless: {
+		id: 'pointless',
+		name: 'Pointless',
+		desc: 'Players get bonus points for guessing the least commonly guessed answers.',
+		onAfterLoad() {
+			this.correct = this.questions.map(() => ({}));
+		},
+
+		onCorrectAnswer(player: ScavengerHuntPlayer, value: string) {
+			const curr = player.currentQuestion;
+
+			if (!this.correct[curr][value]) this.correct[curr][value] = [];
+			if (this.correct[curr][value].includes(player.id)) return;
+
+			this.correct[curr][value].push(player.id);
```

Comment:
Could you add some way to score the formatting of the original answer? Currently this bit will show the ID while the hunt itself shows the formatted answer.

(Not a very required change; is fine to skip if this would take too much effort)